### ISSUE TITLE
Accessibility Issues with MultiSelect.Filterable

### ISSUE THREAD

**Issue by jimmywu1** [carbon-design-system/carbon#3074]
## What package(s) are you using?

- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

Describe in detail the issue you're having. Is this a feature request (new
component, new icon), a bug, or a general issue?

> Is this issue related to a specific component?
Yes. `MultiSelect.Filterable`

> What browser are you working in?
Version 74.0.3729.169 (Official Build)

> What version of the Carbon Design System are you using?
v9

> Tool used: DAP

---

**Violation 1:** The input element has the id "downshift-0-item-0" that is empty or already in use.

<img width="509" alt="Screen Shot 2019-06-14 at 8 20 19 AM" src="https://user-images.githubusercontent.com/51823915/59508830-43bc9780-8e7d-11e9-85b2-a5e43dd11cd1.png">

---

**Violation 2:** The id "downshift-0-item-0" referenced by the 'for' attribute of a label element is not a valid input element.

This probably occurred because it detected the id "downshift-0-item-0" on the div first.

---

**Violation 3:** The expanded combobox does not own or control a valid popup type.
<img width="490" alt="Screen Shot 2019-06-14 at 8 23 18 AM" src="https://user-images.githubusercontent.com/51823915/59509022-d9582700-8e7d-11e9-9515-a7676949554d.png">

---

**Potential Violation:** An element with WAI-ARIA role listbox does not contain or own at least 1 child element with each of the following WAI-ARIA roles: option
<img width="526" alt="Screen Shot 2019-06-14 at 8 16 20 AM" src="https://user-images.githubusercontent.com/51823915/59509085-099fc580-8e7e-11e9-98bf-9ba0492fd564.png">

**Comment by dakahn** [comment]
Not seeing this issue in my latest testing. Happy to reopen if I need to 👍

**Comment by jamesjmcconnell** [comment]
I still see this behaviour in carbon-components-react v6.112,  could you reopen this issue and take a look again @dakahn ?

**Comment by dakahn** [comment]
Missed that we're talking about legacy Carbon here. Reopened and tagged correctly.

**Comment by tw15egan** [comment]
Just an FYI, we aren't currently resolving any accessibility issues for v6 of carbon-components-react but you are welcome to contribute a PR and we will work quickly on getting it merged in

**Comment by kglickman** [comment]
I'm seeing a DAP issue with
		"carbon-components": "10.14.0",
		"carbon-components-react": "7.14.0",
that I can also reproduce in the story book.
![image](https://user-images.githubusercontent.com/30597411/87190197-0ca6f880-c2c0-11ea-8320-45e68e529a16.png)

**Comment by kglickman** [comment]
@tw15egan The [issue](https://github.com/carbon-design-system/carbon/issues/3074#issuecomment-656846923) I reported just above is with version 10.  This defect doesn't seem active and is tagged with version 9.  Should I create a new one against 10?

**Comment by tw15egan** [comment]
@kglickman that would be really helpful! I'll close this one and properly triage the new one once created 👍

**Comment by kglickman** [comment]
@tw15egan New defect is https://github.com/carbon-design-system/carbon/issues/6570.
